README.md

```diff
@@ -1 +1,44 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse
+
+The purpose of this project is to create a basic single-page Hello Microverse. you are required to set up Html-CSS linter according to the guidelines specified in the Html-CSS limter config instructions, create the website using a mobile first, run.
+
+## Built With
+
+- HTML5
+- CSS3
+- Linters
+
+
+
+To get a local copy .
+
+### Prerequisites
+- know how to use figma
+- Have VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Install node package. [Link to download node](https://nodejs.org/en/download/)
+- Have git installed.[Link to download git](https://git-scm.com/downloads)
+- Create a github repository.
+- On your local machine run the following git commnad to clone my repository
+- git clone <my github repository url>
+
+
+## Author
+
+👤 **Christoper abraham**
+
+- GitHub: [@githubhandle](https://github.com/Cabraham1)
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abrahamchristopher)
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Please kindly use the updated version of the [Microverse README template](https://github.com/microverseinc/readme-template) and customize it accordingly, all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present your repo.